nsqd/nsqd_test.go

```diff
@@ -23,8 +23,15 @@ const (
 	RequestTimeout = 5 * time.Second
 )
 
+func newMetadataFile(opts *Options) string {
```

Comment:
Can you move these two functions into `nsqd.go` and replace the two separate uses there?